examples/rossmann-store-sales-example.ipynb

```diff
@@ -55,21 +55,22 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/opt/conda/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: \n",
+      "/opt/conda/envs/rapids/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: \u001b[1m\n",
       "Environment variables with the 'NUMBAPRO' prefix are deprecated and consequently ignored, found use of NUMBAPRO_NVVM=/usr/local/cuda/nvvm/lib64/libnvvm.so.\n",
       "\n",
-      "For more information about alternatives visit: ('https://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')\n",
+      "For more information about alternatives visit: ('http://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')\u001b[0m\n",
       "  warnings.warn(errors.NumbaWarning(msg))\n",
-      "/opt/conda/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: \n",
+      "/opt/conda/envs/rapids/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: \u001b[1m\n",
       "Environment variables with the 'NUMBAPRO' prefix are deprecated and consequently ignored, found use of NUMBAPRO_LIBDEVICE=/usr/local/cuda/nvvm/libdevice/.\n",
       "\n",
-      "For more information about alternatives visit: ('https://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')\n",
+      "For more information about alternatives visit: ('http://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')\u001b[0m\n",
       "  warnings.warn(errors.NumbaWarning(msg))\n"
      ]
     }
    ],
    "source": [
     "import os\n",
+    "os.environ[\"CUDA_VISIBLE_DEVICES\"]=\"2\"\n",
```

Comment:
We should leave CUDA_VISIBLE_DEVICES out of notebooks (especially since this breaks us on systems with less than 3 gpu's).